com.itopia.client.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would be nice to have the appdata and desktop files included in the zip file in the future.